kolibri/plugins/device/assets/src/views/ManageContentPage/ChannelPanel/WithSizeAndOptions.vue

```diff
@@ -110,4 +154,23 @@
     margin: 0;
   }
 
+  .private-icons {
+    display: inline-block;
+    margin-top: -3px;
+    margin-bottom: 3px;
+    vertical-align: top;
+
+    .panel-sm & {
+      margin-top: -1px;
+      margin-bottom: 1px;
+    }
+  }
+
+  .new-label {
+    padding: 2px 5px 2px 4px;
+    margin-left: 8px;
+    font-size: 14px;
+    border-radius: 2px;
```

Comment:
recommend some slight styling changes to this and the other 'New' label:

* bold
* 8px left and right padding
* align text bottom with row


| before | after |
|--|--|
| ![image](https://user-images.githubusercontent.com/2367265/70960787-df99ac80-2034-11ea-8bd4-10b0cd54f00d.png)|![image](https://user-images.githubusercontent.com/2367265/70960735-b547ef00-2034-11ea-9b0c-d8e9cbcd27e4.png)|

```css
  .new-label {
    position: relative;
    top: 2px;
    padding: 2px 8px;
    margin-left: 8px;
    font-size: 14px;
    font-weight: bold;
    border-radius: 2px;
  }
```